packages/react-app/src/components/filter.tsx

```diff
@@ -37,10 +37,12 @@ export function Filter() {
           onChange={(val: any) => setFilter({ ...filter, expire: val })}
         />
       </Field>
-      <Field label="Show expired">
-        <Switch
-          checked={filter.showExpired}
-          onChange={(val: any) => setFilter({ ...filter, showExpired: val })}
+      <Field label="Show status">
+        <DropDown
+          id="filterDropdown"
+          items={['Active', 'Expired']}
+          selected={filter.showStatus}
```

Comment:
-We should change it to status as a string in FilterModel.
-Set ENUM_QUEST_STATE.Active as default value in DEFAULT_FILTER
the dropdown will then attribute the corresponding status string on selected and is supposed to select Active on page load without doing anything else.